OracleDataIntegrator/README.md

```diff
@@ -59,7 +59,7 @@ Download the Oracle Server JRE binary and drop in folder `OracleJDK/java-8` and
         $ cd OracleJDK/java-8
         $ sh build.sh
 
-**NOTE:** The files to build JDK image can be found under [docker/OracleJDK](https://github.com/oracle/docker-images/tree/master/OracleJDK)
+**NOTE:** The files to build JDK image can be found under [docker/OracleJDK](https://github.com/oracle/docker-images/tree/master/OracleJDK) or image can be pulled from OCR or DockerStore
```

Comment:
The Oracle Java Server JRE is available on the Container Registry and Docker Store. As above, please provide links.